pynxtools_xps/file_parser.py

```diff
@@ -24,10 +24,9 @@
 from typing import List, Dict
 
 from pynxtools_xps.sle.sle_specs import SleMapperSpecs
-from pynxtools_xps.txt.txt_scienta import TxtMapperScienta
+from pynxtools_xps.scienta.scienta_txt import TxtMapperScienta
```

Comment:
I liked this approach to have the sub-package according to the software instrument.

Reply:
Yeah, I will change this slowly when I update the other parsers as well, e.g., group all of the specs readers.